global-control-plane.md

```diff
@@ -0,0 +1,284 @@
+
+```bash
+oc create route passthrough kong-mesh-control-plane --service=kong-mesh-control-plane --port=https-api-server 
+```
+
+Check it is working
+
+```bash
+$ http --verify=false https://`oc get route kong-mesh-control-plane -n kong-mesh-system -ojson | jq -r .spec.host`/config | jq .mode
+"global"
+```
+
+Create the Global Remote Sync route and extract the endpoint:
+
+```bash
+oc create route passthrough kong-mesh-global-zone-sync --service=kong-mesh-global-zone-sync --port=global-zone-sync -n kong-mesh-system
+export GLOBAL_SYNC=$(oc get route -n kong-mesh-system kong-mesh-global-zone-sync --template='{{ .spec.host }}'):443
+```
+
+### Install the Mesh Control Plane in Zone 1
+
+Login to the `kong-dp1` cluster
+
+```bash
+kumactl install control-plane --cni-enabled --mode=zone --zone=kumadp1 --ingress-enabled --kds-global-address grpcs://${GLOBAL_SYNC} | oc apply -f -
+```
+
+Check the zone is discovered in the global. Connect to the global control plane cluster to get the route
+
+```bash
+$ http `oc get route kong-mesh-control-plane -n kong-mesh-system -ojson | jq -r .spec.host`/status/zones                            
+HTTP/1.1 200 OK
+cache-control: private
+content-length: 48
+content-type: application/json
+date: Fri, 03 Jun 2022 15:54:07 GMT
+set-cookie: 559045d469a6cf01d61b4410371a08e0=a2d29fef296bbefe83902b97d4268a00; path=/; HttpOnly
+
+[
+    {
+        "active": true,
+        "name": "kumadp1"
+    }
+]
+```
+
+### Install Kong Gateway DP
+
+- create ns kong-dp
+
+```bash
+oc create ns kong-dp
+```
+
+- create license secret
+
+```bash
+oc create secret generic kong-enterprise-license --from-file=license=./license.json -n kong-dp
+```
+
+- create certificate secret using the existing same certs we used in the control plane
+
+```bash
+oc create secret tls kong-cluster-cert --cert=./cluster.crt --key=./cluster.key -n kong-dp
+```
+
+- add the serviceaccount to the anyuid scc so the sidecars can be created and annotate the namespace. 
+That will allow the pod to be created together with the sidecar
+
+```bash
+oc adm policy add-scc-to-group anyuid system:serviceaccounts:kong-dp
+kubectl label namespace kong-dp kuma.io/sidecar-injection=enabled
+```
+
+- install the Data Plane. Use the appropriate endpoints depending on the Control plane install (VM/OCP)
+
+```bash
+sed -e 's/\$CLUSTER_URL/'"$CLUSTER_URL"'/' -e 's/\$CLUSTER_TELEMETRY_URL/'"$CLUSTER_TELEMETRY_URL"'/' gateway-multizone/dp-values.yaml | helm install kong -n kong-dp kong/kong -f -
+```
+
+- expose proxy routes (http/https)
+
+```bash
+oc expose svc kong-kong-proxy -n kong-dp --port=kong-proxy
+oc create route passthrough kong-kong-proxy-tls --port=kong-proxy-tls --service=kong-kong-proxy -n kong-dp
+```
+
+- check clustering
+
+**VM installation**: Refer to [Verify clustering is working](./gateway-multizone/kong-vm.md#verify-clustering-is-working)
+
+```bash
+http `oc get route -n kong kong-kong-admin --template='{{ .spec.host }}'`/clustering/status
+```
+
```

Comment:
You need to use a tools like kubectx to change the context.

Reply:
Thanks for the suggestion